README.md

```diff
@@ -379,9 +380,15 @@ Uncertainty predictions may be calibrated to improve their performance on new pr
 * `tscaling` Similar to zscaling. Assumes that the errors are normally distributed, but accounts for the ensemble size and uncertainty in the sample variance by using a sample-size reduced t-distribution in the negative log likelihood. Works best when errors are mostly due to variability between model instances and not dataset noise or model bias.
 * `zelikman_interval` Assumes that the error distribution is the same for each prediction but scaled by the uncalibrated standard deviation for each. Multiplies the uncalibrated standard deviation by a factor necessary to cover the specified interval of the calibration set. Does not assume a Gaussian distribution. Intended for use with intervals but can return a stdev as well. (https://arxiv.org/abs/2005.12496)
 * `mve_weighting` For use with ensembles of models trained with mve or evidential loss function. Uses a weighted average of the predicted variances to achieve a minimum negative log likelihood of predictions. (https://doi.org/10.1186/s13321-021-00551-x)
+* `conformal_regression` Generates an interval of fixed size for each prediction such that the actual value has probability $1-\alpha$ of falling in the interval. The desired error rate is controlled using the parameter `--alpha <float>` which is set by default to 0.1. (https://arxiv.org/abs/2107.07511)
+* `conformal_quantile_regression` Generates an interval for each prediction based on quantile regressions of the data. The desired error rate is controlled using the parameter `--alpha <float>` which is set by default to 0.1. The loaded model should output quantile predictions. (The trained model should output $\alpha/2$ and $1-\alpha/2$ quantiles.) Such a model model can be trained with parameters `--loss_function quantile_interval` and `--alpha <float>` where $\alpha$ is the desired error rate. (https://arxiv.org/abs/2107.07511)
```

Comment:
I don't understand the difference between these two from the description. Is `conformal_regression` a symmetric interval and `conformer_quantile_regression` not required to be symmetric? These could use names that more clearly distinguish them.

Does only the latter one require the special loss function?

Reply:
Hey Charles, I rewrote this section to make it more clear what the difference is. As for the names, I think it would make more sense to use the names given in the linked paper.